src/components/interface/StepsPanel.tsx

```diff
@@ -123,6 +123,12 @@ function StepItem({
   const studyNavigateTo = () => (participantView ? navigate(`/${studyId}/${stepIndex}`) : navigate(`/${studyId}/reviewer-${step}`));
   const navigateTo = analysisNavigation ? analysisNavigateTo : studyNavigateTo;
 
+  // eslint-disable-next-line no-nested-ternary
+  const coOrComponents = step.includes('.co.')
+    ? 'co.'
+    : (step.includes('.components.') ? '.components.' : false);
+  const cleanedStep = step.includes('$') && coOrComponents && step.includes(coOrComponents) ? step.split(coOrComponents).at(-1) : step;
+
```

Comment:
Are you basically allowing a user to use "$test.components.markdown-component" OR "$test.co.markdown-component"?? If so, is this necessary? Also, should the check have '.co.' on line 128? Don't think it actually ends up affecting the final cleanedStep, but at least for clarity.

Reply:
Fixed in [6ac848e](https://github.com/revisit-studies/study/pull/488/commits/6ac848e580ad73fe31ed9e0439720d02bf4c62a8)